app/models/form/sales/questions/shared_ownership_type.rb

```diff
@@ -37,6 +44,10 @@ def answer_options
   end
 
   def guidance_partial
-    "shared_ownership_type_definitions" if form.start_date.year >= 2023
+    if form.start_year_after_2024?
+      "shared_ownership_type_definitions_2024"
+    elsif form.start_date.year >= 2023
```

Comment:
would == 2023 be clearer here?

Reply:
I kept it as this cause that's what it already was, I don't think it makes much of a difference 🤷